tests/compile-fail/stacked_borrows/illegal_read6.rs

```diff
@@ -1,8 +1,10 @@
 // Creating a shared reference does not leak the data to raw pointers.
-fn main() { unsafe {
-    let x = &mut 0;
-    let raw = x as *mut _;
-    let x = &mut *x; // kill `raw`
-    let _y = &*x; // this should not activate `raw` again
-    let _val = *raw; //~ ERROR borrow stack
-} }
+fn main() {
+    unsafe {
```

Comment:
I'm not a fan of the double-indentation rustfmt introduces here... but I guess it doesn't really matter since these are just tests.